MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -1399,9 +1401,8 @@
 // Populates the Metal lookup maps
 void MVKPixelFormats::buildMTLFormatMaps() {
 
-	// Set all MTLPixelFormats and MTLVertexFormats to undefined/invalid
+	// Set all MTLPixelFormats lookups to undefined/invalid
```

Comment:
```suggestion
	// Set all MTLPixelFormat lookups to undefined/invalid
```

Reply:
Fixed. This line is no longer necessary.